### ISSUE TITLE
Submission Review Results

### ISSUE THREAD

**Issue by lilasaba** [lilasaba/medilatin#14]
# SUBMISSION REVIEW RESULTS
#### Amount of rewriting required:	2

**Main contributions**:
The paper presents the efforts of digitizing medieval Latin charter data using a target language independent speech recognizer. An acoustic model for medieval Latin was built using speech data from different source languages of the Visegrad region. To build an acoustic model without source language speech data, two grapheme-based pronunciation modelling approaches are proposed. In the first approach, source-language phonemes were mapped to target-language phonemes using the expert knowledge that was implemented as a set of context independent digraph mappings and context dependent rewrite rules. In the second approach, the four-language unified simplified grapheme acoustic model was used, where all special characters were mapped to their normalized form, and those graphemes that are non-native to Latin, and can straightforwardly mapped to a native Latin grapheme or several such graphemes, were replaced. The experimental results show that the speech recognition systems based on the both proposed modelling methods outperform the baseline system. Some unexpected results are reported as well that indicate that the four-language unified simplified grapheme acoustic model is able to generalize better that the other presented models.
**Positive aspects:**
The positive aspect of the presented work is that it represents an important contribution to the efforts of digitizing medieval Latin charter data. An important finding is also that the unified simplified grapheme acoustic model can yield better results that the models that are based on source-language phonemes to target-language phonemes mappings.
**Negative aspects:**
It is hard to understand from the paper how the differences between the four languages are actually reflected in the two grapheme-based pronunciation modelling approaches.
**Reviewer's comments:**
It would be useful if some explicit examples of pronunciations are given in the paper that would illustrate the difference between the two grapheme-based pronunciation modelling approaches.

#### Amount of rewriting required:	1
**Main contributions:**
The authors describe 2 approaches for the building of a speech recognition components, namely acoustic model, for a dictation scenario for the Latin langauge, spoken in the Visegrad region. The use existing databases with largely varying amounts of transcribed data (Czech, Hungarian, Polish and by far the most data for Hungarian). They provided results for mono-lingual baseline systems for these languages. Approach 1) they mapped the respective phonemes to a "Latin" phoneme set. Approach 2) they mapped the grahemes to a "unified" (simplified) "grapheme" set and build acoustic models with data from 3 or 4 languages respectively. Both approaches outperform the baseline mono-lingual models. They mentioned the relevance of the different amount of training data per languages.
**Positive aspects:**
The authors describe well the available databases, the building essentials for the acoustic model and provided numerous results for the two investigated approaches including reasoning. As the writing systems of these language are based on Latin they chose to train acoustic models using "graphemes" as subwird unit.
**Negative aspects:**
The database for Hungerian is much bigger then the other language databases, thus more robust models are to be expected. The 3 or 4 languages acoustic models show more stabalized results as expected. It would be interesting to verify that a "unified phoneme" approach for using 3 or 4 languages as databases would give similiar results like the "grapheme" approach. Clearyl the "grapheme" approach would be easier to handle in real use, as no G2P converter will need to be trained.
**Reviewer's comments:**
The English should be slighly revised for typos.

**Comment by lilasaba** [comment]
@mihajlik, @balogandras, @btarjan, úgy tűnik elfogadták, feljebb közlöm a review-kat változatlanul.

**Comment by lilasaba** [comment]
Camera-ready verzio feltoltve.
